### DISCUSSION TITLE
Go back to a certain date in tags using timestamps

### DISCUSSION THREAD

**Discussion by vfcdsjn** [AprilSylph/XKit-Rewritten#302] [Suggestion Box]
Hello, thank you for bringing us Xkit-Rewritten.

Something that tumblr's been sorely lacking since the implementation of the new interface is the ability to go back to a certain date in tags using timestamps. Xkit didn't need this feature back then because tumblr already had it but now that they have taken it away is it possible for you guys to add a feature like this to Xkit-Rewritten? Please and thank you very much.

Stay safe!

**Comment by AprilSylph** [comment]
This is planned; see #266

**Reply by vfcdsjn** [reply]
I see, will you be working on implementing it to dashblogs or will it also work on tag searches such as: tumblr.com/tagged/one+piece

**Reply by AprilSylph** [reply]
The pages it would work is limited by if Tumblr supports skipping using timestamps. I'm actually not sure of the full list of pages which support this, but I believe it's just the Inbox, Likes, and global `/tagged/` pages.

**Reply by AprilSylph** [reply]
Hmm, it actually looks like Tumblr doesn't support this for tagged pages. If Tumblr doesn't support skipping back, there's no feasible way to force it to skip back to older posts.

**Reply by vfcdsjn** [reply]
I see 😔 Thank you for looking into it.
